<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cos_GroupChat</class>
 <widget class="QWidget" name="Cos_GroupChat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cos_GroupChat</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#m_title{
	background-color: rgb(233,233,235);
}
#Cos_GroupChat{
	
	background-color: rgb(255, 255, 255);
}
#m_pframe1{
	background-color: rgb(255, 255, 255);
}
#m_label_title{
	
	font: 12pt &quot;黑体&quot;;
}
#m_widget_bottom{  
	border-style: solid;
  border-width: 2px;
  border-color: rgb(233,233,235);
}
#m_textEdit_send{
	border:0px;
}
#m_Btn_close{
	
	border-image: url(:/NewCos/Resources/close.png);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QWidget" name="m_title" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="m_label_title">
          <property name="text">
           <string>呆呆肉肉的聊天</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>283</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="m_Btn_close">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="m_pframe">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>300</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="m_textEdit_send"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>撤销对方</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>删除我方</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_Btn_Send">
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
